hack/chains/rekor-image-lookup.sh

```diff
@@ -5,8 +5,13 @@ IMAGE_URL=${1:-$DEFAULT_URL}
 DEFAULT_REKOR_SERVER=https://rekor.sigstore.dev
 REKOR_SERVER=${2:-$DEFAULT_REKOR_SERVER}
 
-# In reality we might know the digest already, but let's look it up
-IMAGE_DIGEST=$( skopeo inspect --no-tags docker://$IMAGE_URL | jq -r .Digest )
+if [[ $IMAGE_URL =~ ^(sha256:)?[0-9a-f]+$ ]] && [[ ${#IMAGE_URL} > 63 ]]; then
```

Comment:
I wonder if this will work in bash: `^(sha256:)?[0-9a-f]{63}$`

Reply:
Yeah, I tried that, no dice. I'll give it another google, maybe the curlies need escaping or something...

Reply:
Ugh, it works fine. Not sure what I was doing wrong the other day..